client/src/lib/dependency-injection/provideInjector.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Injector from 'lib/Injector';
+
+function provideInjector(Component) {
+  class InjectorProvider extends React.Component {
+    getChildContext() {
+      const { get } = Injector;
+      return {
+        injector: { get },
+      };
+    }
+
+    render() {
+      return <Component {...this.props} />;
+    }
+  }
+
+  InjectorProvider.childContextTypes = {
+    injector: React.PropTypes.shape({
+      get: React.PropTypes.func,
+    }),
+  };
+
+  return InjectorProvider;
+}
+
+export {
```

Comment:
Yeah these should be default exports. They're named exports in the main `Injector` file, and i think this is just legacy.